src/XRP/issued-currency-client.ts

```diff
@@ -689,6 +689,62 @@ export default class IssuedCurrencyClient {
+   *
+   * @param sender The Wallet from which issued currency will be sent, and that will sign the transaction.
+   * @param destination The destination address for the payment, encoded as an X-address (see https://xrpaddress.info/).
+   * @param currency The three-letter currency code of the issued currency being sent.
+   * @param issuer The issuing address of the issued currency being sent, encoded as an X-address.
+   * @param amount The amount of issued currency to pay to the destination.
+   * @param transferFee (Optional) The transfer fee associated with the issuing account, expressed as a percentage. (i.e. a value of .5 indicates
+   *                    a 0.5% transfer fee).  Supply this field for automatic calculation of the sendMax value for this payment.
+   *                    Either this or sendMaxvalue may be specified, but not both.
+   * @param sendMaxValue (Optional) A manual specification of the maximum amount of source currency this payment is allowed to cost,
+   *                      including transfer fees, exchange rates, and slippage. Does not include the XRP destroyed as a cost for submitting
+   *                      the transaction. Either this or transferFee may be specified, but not both.
+   * @param useAnyValidIssuer (Optional) Defaults to false. Whether to invoke the special case of sending currency issued by "any issuer that the destination accepts."
+   *                          This includes all addresses to which the destination has extended trust lines, as well as currencies issued by the destination.
+   *                          See https://xrpl.org/payment.html#special-issuer-values-for-sendmax-and-amount
+   */
+  public async sendIssuedCurrencyPayment(
+    sender: Wallet,
+    destination: string,
+    currency: string,
+    issuer: string,
+    amount: string,
+    transferFee?: number,
+    sendMaxValue?: string,
+  ): Promise<TransactionResult> {
+    if (sender.getAddress() === issuer) {
+      throw new XrpError(
+        XrpErrorType.InvalidInput,
+        'The sending address cannot be the same as the issuing address. To create issued currency, use `createIssuedCurrency`.',
+      )
+    }
+    if (destination === issuer) {
+      throw new XrpError(
+        XrpErrorType.InvalidInput,
+        'The destination address cannot be the same as the issuer. To redeem issued currency, use `redeemIssuedCurrency`.',
+      )
+    }
+    // TODO: (acorso) A payment like this might fail if:
+    // - the issuer has not enabled rippling on its account
+    // - the issuer has enabled authorized trust lines, and the recipient of the payment does NOT have an authorized trust line to the issuer
+    // We may want to decide if it's worth incurring the cost of querying for this information BEFORE sending a payment that will destroy
+    // a transaction cost.  Or maybe we provide a different public method that can perform these checks if the user cares.
```

Comment:
i think you could get away with including this in the docs for now...

might be good to track this with a task somehow as well

Reply:
Removed comment, created a task: https://app.asana.com/0/1190819649053561/1199151116842791/f